.github/workflows/minos.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+name: Python package
+on:
+  [ push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      matrix:
+        python-version: [ 3.8, 3.7 ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Cache conda
+      id: cache-miniconda
+      uses: actions/cache@v1
+      env:
+        CACHE_NUMBER: 0
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        path: ~/conda_pkgs_dir
+        key:
+          ${{ runner.os }}-conda-${{ hashFiles('./environment.yml') }}
+    - name: Install system development tools
+      run: |
+        export DEBIAN_FRONTEND=noninteractive
+        sudo apt update
+        sudo apt install -y build-essential zlib1g-dev zlib1g
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        miniconda-version: "latest"
+        environment-file: ./environment.yml
+        mamba-version: "*"
+        use-mamba: true
+        channels: bioconda, conda-forge, defaults
+        use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+        auto-update-conda: true
+        activate-environment: "test"
+        auto-activate-base: false
+    - name: Install dependencies
+      run: |
+        python setup.py install
```

Comment:
I thought you'd move this to bdist_wheel? Maybe I misunderstood..

Reply:
I made `install` and `bdist_wheel` equivalent. But good point, I will update the test

Reply:
I mean, whichever "works best". In the long run that should be a conda installation.

Reply:
Fair. In the meantime, I think that wheel will do :)